### ISSUE TITLE
Nav item - browser tooltip always showing

### ISSUE THREAD

**Issue by hrc7505** [OfficeDev/office-ui-fabric-react#11413]
### Environment Information

- **Package version(s)**: 7.69.2

### Please provide a reproduction of the bug in a codepen:
[Codepen](https://codepen.io/hardik-chaudhary/pen/MWYyWrZ?editors=1010)

#### Actual behavior:
Nav item tooltip is always visible on mouse hover. Same result with ```title=""```  with passing in NavLink. 

#### Expected behavior:
```title=""``` should not show tooltip. Or, 
There is something that can tells component to show ```title``` or not 

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: High

**Comment by micahgodbolt** [comment]
You just need to pass in a non empty string to the title, otherwise Nav uses the text value.
This works fine.

```js
{ name: 'Sales', icon: 'Financial', key: 'key2', title:" " }
```

**Comment by hrc7505** [comment]
@micahgodbolt I already tried this solution but microsoft edge showing empty tooltip there.
![image](https://user-images.githubusercontent.com/17545364/70593608-b4adf380-1c03-11ea-8685-b44cbc76720d.png)

**Comment by micahgodbolt** [comment]
@hrc7505 ah, good catch. I noted that new edge handled it well, but doesn't seem to be the case for old edge

**Comment by joschect** [comment]
@micahgodbolt and @hrc7505 I think that this is something that we should support. I believe the fix would be to have title take in `null` as a value. Unfortunately that's the only way we can preserve current behavior of falling back to text while allowing an opt out. Should be a quick change that I think makes sense. @micahgodbolt any thoughts?

**Comment by micahgodbolt** [comment]
Yeah, i was hoping there was a non code-change solution here, but yeah. We should be able to be able to support that null value without TOO much regressions. I
